_dashboards/branding.md

```diff
@@ -90,6 +93,57 @@ mark:
 applicationTitle: "My custom application"
 ```
 
+### Condensed header
+
+The condensed header view reduces the footprint of the header and frees up space on the page by combining navigational elements into a single header bar.
+
+The current default view remains close in appearance to the two-bar header offered in the previous version of Dashboards, with minor differences. To specify the condensed header, add the configuration property `useExpandedHeader` to the `opensearch_dashboards.yml` file and set the value to `false`, as the following example illustrates.
+
+ ```yml
+# opensearchDashboards.branding:
+  # logo:
+    defaultUrl: "https://example.com/sample.svg"
+    darkModeUrl: "https://example.com/dark-mode-sample.svg"
+  # mark:
+    # defaultUrl: ""
+    # darkModeUrl: ""
+  # loadingLogo:
+    # defaultUrl: ""
+    # darkModeUrl: ""
+  # faviconUrl: ""
+  applicationTitle: "my custom application"
+  useExpandedHeader: false
+```
+
+In a future release, default behavior will become `useExpandedHeader: false`. If you want to retain the default view in subsequent releases, you can explicitly set the property to `true` in advance. You can also wait to do this when upgrading.
+{: .note }
+
+The condensed view header appears as in the example here.
+
+![Condensed header]({{site.url}}{{site.baseurl}}/images/DBs-Condensed.jpeg)
+
+Header element | Description
+:--- | :---
+OpenSearch logo | See #1. Functions as the home button.
+Header bar | See #2. A single header bar used for all navigation controls.
+
+The default view remains close to the traditional view with minor changes.
+
+![Default header]({{site.url}}{{site.baseurl}}/images/DBs-Traditional.jpeg)
+
+Header element | Description
+:--- | :---
+Home button | See #1. Returns to home page and provides an indication when a page is loading.
+Header label | See #2. The label also functions as a home button.
+Navigation controls | See #3. Additional navigation controls on right-side insertion points.
+
+#### Preserving nagivation elements in default view
```

Comment:
Should we consistently use "the default view" as opposed to using just "default view" here?